db/migrate/20151021234549_add_column_to_email_links.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnToEmailLinks < ActiveRecord::Migration
+  def change
+    add_column :email_links, :read, :boolean
+  end
+end
```

Comment:
since all these migrations are in the same pull request, it's safe to condense them into one migration. As far as I can tell, the goal here was to 

1. provide null constraints on a few columns
2. change the name of the table from `emaillinks` to `email_links`
3. add the `read` column as a boolean column

While it might make sense to do this over many migrations in development, it's helpful to combine the migrations into one migration for the merge into master (especially when they're all very related as these are). It might seem weird, but as a project's code base grows, migration files can get out of hand and maintaining them for new developers (and new environments) gets more complex.

For this merge into master, neither `email_links` or `emaillinks` tables exist in production yet so it should be easy. First, migrate down this version by issuing the command:

`rake db:migrate:down VERSION=20151021234549`

and

`rake db:migrate:down VERSION=20151012192529`

Then add this migration's column definition to 20151012192529. You can delete migration 20151012184419 and add the `drop_table :emaillinks` above the create table block.